initrd/bin/kexec-seal-key

```diff
@@ -136,7 +137,11 @@ tpmr pcrread -a 7 "$pcrf"
 
 DO_WITH_DEBUG --mask-position 7 \
 	tpmr seal "$KEY_FILE" "$TPM_INDEX" 0,1,2,3,4,5,6,7 "$pcrf" \
-	"$TPM_SIZE" "$key_password"
+	"$TPM_SIZE" "$key_password" || {
+	shred -n 10 -z -u /tmp/secret/tpm_owner_password 2>/dev/null
```

Comment:
@JonathonHall-Purism I understand that there are two cases where the tpm owner passphrase can be used and if entered wrongly, will fail and cache of it should be invalidated. This is one of them

Reply:
Thread on seal-totp addresses this, resolving